packages/flutter_goldens_client/lib/skia_client.dart

```diff
@@ -121,10 +121,15 @@ class SkiaGoldClient {
         .path,
     ];
 
-    await io.Process.run(
+    final io.ProcessResult result = await io.Process.run(
       _goldctl,
       authArguments,
     );
+
+    if (result.exitCode != 0) {
```

Comment:
Should this actually throw to indicate that there was an error instead of silently swallowing it? Here and below.

Reply:
We don't throw on master to prevent the tree from going red when someone is landing a change.